### PR TITLE
Add Release CI

### PR DESCRIPTION
**Pull request by fewensa**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -33,21 +37,74 @@ jobs:
...
+          mkdir -p shared/crab-parachain-runtime
+          mkdir -p shared/darwinia-parachain-runtime
+          mkdir -p shared/polkadot-runtime
           zip -jr \
             shared/darwinia-collator-linux-x86_64.zip \
             target/x86_64-unknown-linux-gnu/release/darwinia-collator
 
+          mv target/x86_64-unknown-linux-gnu/release/wbuild/crab-parachain-runtime/*.wasm shared/crab-parachain-runtime
+          mv target/x86_64-unknown-linux-gnu/release/wbuild/darwinia-parachain-runtime/*.wasm shared/darwinia-parachain-runtime
+          mv target/x86_64-unknown-linux-gnu/release/wbuild/polkadot-runtime/*.wasm shared/polkadot-runtime
```
Comment (fewensa) [inline .github/workflows/release.yml]: Is this required (polkadot-runtime)?

Comment (acgxv) [reply .github/workflows/release.yml]: Don't need Polkadot runtime.

**Comment by fewensa** [comment]
Staging
https://github.com/darwinia-network/darwinia-parachain/actions/runs/1743842434

Production
https://github.com/fewensa/darwinia-parachain/actions/runs/1744067657
https://github.com/fewensa/darwinia-parachain/releases/tag/v5.0.2

Please wait for https://github.com/fewensa/darwinia-parachain/actions/runs/1744379262 check release template is correct

**Comment by fewensa** [comment]
https://github.com/fewensa/darwinia-parachain/releases/tag/v5.0.3

Done.

File: .github/release-template.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| :----------------: | :------------: | :--------------: |
+| Darwinia Parachain |       2        |       HIGH       |
+|  Crab  Parachain   |       2        |       HIGH       |
+
+## Resources
+
+### Pre-built Binary
+|  OS   |  Arch  | Glibc | LLVM  |                                                                 Download                                                                 |
+| :---: | :----: | :---: | :---: | :--------------------------------------------------------------------------------------------------------------------------------------: |
+| Linux | x86_64 | 2.23  |  4.0  | [tar.bz2](https://github.com/darwinia-network/darwinia-parachain/releases/download/{{ tag }}/darwinia-collator-x86_64-linux-gnu.tar.bz2) |
```
Comment (acgxv) [inline .github/release-template.md:18]: Please check this.

File: .github/release-template.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+```
+#### Pull with the Git Commit SHA
+```docker
+docker pull quay.io/darwinia-network/darwinia-collator:sha-{{ sha }}
+```
+
+## Proposal Hashes
+| Network  |               Proposal Hash               |
+| :------: | :---------------------------------------: |
+| Darwinia | {{ darwinia_parachain_proposal_compact }} |
```
Comment (acgxv) [inline .github/release-template.md]: Please use compressed WASM.

File: .github/workflows/staging.yml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    container:
+      image: ubuntu:16.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          apt update -y
+          apt install --no-install-recommends -y \
+            curl ca-certificates make gcc g++ clang-4.0 pkg-config zip
```
Comment (acgxv) [inline .github/workflows/staging.yml]: Still found zip usage here.

**Review by acgxv — CHANGES_REQUESTED** [review]

File: .github/release-template.md (hunk 2 above)
Comment (fewensa) [reply .github/release-template.md:18]: https://github.com/fewensa/darwinia-parachain/releases/download/v5.0.3/darwinia-collator-x86_64-linux-gnu.tar.bz2

https://github.com/darwinia-network/darwinia-parachain/releases/download/{{ tag }}/darwinia-collator-x86_64-linux-gnu.tar.bz2

It's  right

**Comment by fewensa** [comment]
Staging: https://github.com/fewensa/darwinia-parachain/actions/runs/1759773634
Release: https://github.com/fewensa/darwinia-parachain/actions/runs/1759773868

**Comment by fewensa** [comment]
https://github.com/fewensa/darwinia-parachain/releases/tag/v5.0.4
All tests passed

**Pull request merged**